src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -1375,9 +1376,9 @@ internal virtual bool CanAddProjectReference(ProjectId referencingProject, Proje
         /// <exception cref="NotSupportedException">Thrown if the solution contains changes not supported according to the
         /// <see cref="CanApplyChange(ApplyChangesKind)"/> method.</exception>
         public virtual bool TryApplyChanges(Solution newSolution)
-            => TryApplyChanges(newSolution, new ProgressTracker());
+            => TryApplyChanges(newSolution, CodeAnalysisProgress.Null);
 
-        internal virtual bool TryApplyChanges(Solution newSolution, IProgressTracker progressTracker)
+        internal virtual bool TryApplyChanges(Solution newSolution, IProgress<CodeAnalysisProgress> progressTracker)
```

Comment:
note: this is why i couldn't call this object `CodeActionProgress`.  We use this progress concept in more places than code actions, including into our core workspace.  So CodeAnalysisProgress seemed like a good name instead.

Reply:
Also, this will likely be something we want to make public in the future as well.